Ceres, Eris, Ice Asteroids, and the Kuiper Belt (#193)
endless-sky/Endless-Sky-Delta: data/map planets.txt

	spaceport `The constant stream of freighters passing through this spaceport is a testament to how much food is needed to feed the densely populated Kimek worlds, in particular their homeworld in the neighboring star system. The city that hosts the spaceport has also become something of a tourist destination for those with adventurous culinary tastes; restaurants here serve recipes native to each of the Coalition species, along with some dishes that blend their three cuisines in unique and sometimes surprising ways.`
	security 0.5

planet Ceres
	attributes mining moon "near earth"
	landscape land/snow37

> 	description `Once faintly and occasionally visible from Earth, Ceres enjoyed status as a planet in the ninteeth century prior the discovery of Neptune and the asteroid belt. Its status wavered, usually considered an asteroid, until the discovery of Eris which led to its classification as a dwarf planet.`

Reviewer: ```suggestion
	description `Once faintly and occasionally visible from Earth, Ceres enjoyed status as a planet in the nineteeth century prior the discovery of Neptune and the asteroid belt. Its status wavered, usually considered an asteroid, until the discovery of Eris, which led to its classification as a dwarf planet.`
```

Reviewer 2: ```suggestion
	description `Once faintly and occasionally visible from Earth, Ceres enjoyed status as a planet in the nineteenth century prior the discovery of Neptune and the asteroid belt. Its status wavered, usually considered an asteroid, until the discovery of Eris which led to its classification as a dwarf planet.`
```

Spelling of nineteenth in both original and first suggested change. (there were two spelling mistakes in the same word, the first suggested change only fixed the first one)

> 	description	`	Ceres's clay crust over a muddy core sports brine cryovolcones and a tenuous water vapour atmosphere. Conveniently located in the asteroid belt and just within Sol's hyper limit, Ceres has long been a favored stopover for miners and interstellar travellers.`

Reviewer: ```suggestion
	description `	Ceres's clay crust over a muddy core sports brine cryovolcones and a tenuous water vapor atmosphere. Conveniently located in the asteroid belt and just within Sol's hyper limit, Ceres has long been a favored stopover for miners and interstellar travellers.`
```
What is a "cryovolcone"?

Author: That would be a misspelling of cryovolcano.

```suggestion
	description	`	Ceres's clay crust over a muddy core sports brine cryovolcanoes and a tenuous water vapor atmosphere. Conveniently located in the asteroid belt and just within Sol's hyper limit, Ceres has long been a favored stopover for miners and interstellar travellers.`
```

> 	description	`	Ceres's clay crust over a muddy core sports brine cryovolcanoes and a tenuous water vapor atmosphere. Conveniently located in the asteroid belt and just within Sol's hyper limit, Ceres has long been a favored stopover for miners and interstellar travellers.`

Reviewer: Something you missed from my earlier suggestion:
```suggestion
	description `	Ceres's clay crust over a muddy core sports brine cryovolcanoes and a tenuous water vapor atmosphere. Conveniently located in the asteroid belt and just within Sol's hyper limit, Ceres has long been a favored stopover for miners and interstellar travellers.`
```
```suggestion
	description	`	Ceres's clay crust over a muddy core sports brine cryovolcanoes and a tenuous water vapor atmosphere. Conveniently located in the asteroid belt and just within Sol's hyper limit, Ceres has long been a favored stopover for miners and interstellar travellers.`
```
